src/mfd_crossbow.cpp

```diff
@@ -0,0 +1,133 @@
+#include "mfd_crossbow.h"
+#include "common/mavlink.h"
+
+//Basic UAV Parameters
+uint8_t system_id = 1;        // MAVLink system ID. Leave at 0 unless you need a specific ID.
+uint8_t component_id = 1;     // Should be left at 0. Set to 190 to simulate mission planner sending a command
+uint8_t system_type = 1;      // UAV type. 0 = generic, 1 = fixed wing, 2 = quadcopter, 3 = helicopter
+uint8_t autopilot_type = 3;   // Autopilot type. Usually set to 0 for generic autopilot with all capabilities
+uint8_t system_mode = 64;     // Flight mode. 4 = auto mode, 8 = guided mode, 16 = stabilize mode, 64 = manual mode
+uint32_t custom_mode = 0;     // Usually set to 0          
+uint8_t system_state = 4;     // 0 = unknown, 3 = standby, 4 = active
+uint32_t upTime = 0;          // System uptime, usually set to 0 for cases where it doesn't matter
+
+// Flight parameters
+int16_t heading = 0;    // Geographical heading angle in degrees
+float lat = 0.0;        // GPS latitude in degrees (example: 47.123456)
+float lon = 0.0;        // GPS longitude in degrees
+float alt = 0.0;        // Relative flight altitude in m
+float groundspeed = 0.0; // Groundspeed in m/s
+
+// GPS parameters
+int16_t gps_sats = 0;     // Number of visible GPS satellites
+int32_t gps_alt = 0.0;    // GPS altitude (Altitude above MSL)
+float gps_hdop = 100.0;   // GPS HDOP
+uint8_t fixType = 3;      // GPS fix type. 0-1: no fix, 2: 2D fix, 3: 3D fix
+
+void
+MFDCrossbow::send_heartbeat(uint8_t system_id, uint8_t component_id, uint8_t system_type, uint8_t autopilot_type, uint8_t system_mode, uint32_t custom_mode, uint8_t system_state)
+{
```

Comment:
Since all these parameters are constant we don't need to pass them to this function